src/DynamoCoreWpf/Properties/Resources.en-US.resx

```diff
@@ -2239,4 +2239,10 @@ Uninstall the following packages: {0}?</value>
   <data name="NodeTooltipOriginalName" xml:space="preserve">
     <value>Original node name: </value>
   </data>
+  <data name="RenderingMemoryOutageDescription" xml:space="preserve">
+    <value>Please check if the amount of generated geometry is correct. If it is, it might be necessary to disable preview for some nodes before reactivating the background 3D preview.</value>
```

Comment:
Isn't it kind of a contradiction to ask to `consider generating fewer geometry objects` after saying `If it is necessary to create them`? Also this message does not really mention the hint of switching off preview for some nodes, which should be useful.

Reply:
I meant to leave the part about disabling preview from your original message, which is what the `...` (ellipses) indicates. I just thought that instead of geometry is not "correct" you should mention that the amount of geometry created is large.

Reply:
Oh, I see. The message actually says `amount of generated geometry is correct`. I meant to say that the amount may be large by accident. For instance, the user could have inadvertently enabled the `repeat` parameter of `List.Combinations` when it was not needed.